src/components/MultiHopTrade/components/LimitOrder/LimitOrderList.tsx

```diff
@@ -0,0 +1,102 @@
+import type { CardProps } from '@chakra-ui/react'
+import {
+  Card,
+  CardBody,
+  CardHeader,
+  Tab,
+  TabList,
+  TabPanel,
+  TabPanels,
+  Tabs,
+} from '@chakra-ui/react'
+import { foxAssetId } from '@shapeshiftoss/caip'
+import type { FC } from 'react'
+import { usdcAssetId } from 'test/mocks/accounts'
+import { Text } from 'components/Text'
+
+import { LimitOrderCard } from './components/LimitOrderCard'
+import { LimitOrderStatus } from './types'
+
+type LimitOrderListProps = {
+  isLoading: boolean
+} & CardProps
+
+const textColorBaseProps = {
+  color: 'text.base',
+}
+
+export const LimitOrderList: FC<LimitOrderListProps> = ({ isLoading, ...cardProps }) => {
```

Comment:
non-blocking note: we've been putting chakra props as a field in the component props to prevent this - it causes rerenders as every render creates a new set of props via the spread. Our current go-to appraoch would be

```typescript
type LimitOrderListProps = {
  isLoading: boolean
  cardProps: CardProps
}
```

instead of
```
type LimitOrderListProps = {
  isLoading: boolean
} & CardProps
```

Reply:
I like it, ty: https://github.com/shapeshift/web/pull/8020/commits/5f3dcfff44420e135e6223df812707884eb907a1